client/src/index.js

```diff
@@ -1,8 +1,257 @@
-import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import React from 'react';
+import './Login.css';
+
+//from material ui
+import Grid from '@material-ui/core/Grid';
+import { makeStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import Link from '@material-ui/core/Link';
+import Snackbar from '@material-ui/core/Snackbar';
+
+const styles = makeStyles(theme => ({
+    root: {
+        height: '100vh',
+    },
+    paper: {
+        margin: theme.spacing(25, 4),
+        isplay: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+    },
+    image: {
+        backgroundImage: 'url(https://images.unsplash.com/photo-1493770348161-369560ae357d?ixlib=rb-1.2.1&auto=format&fit=crop&w=1000&q=80)',
+        backgroundRepeat: 'no-repeat',
+        backgroundSize: 'cover',
+        backgroundPosition: 'center',
+    },
+    button: {
+        margin: theme.spacing(3, 0, 2),
+    },
+    form: {
+        width: '100%', // Fix IE 11 issue.
+        marginTop: theme.spacing(1),
+    },
+  }));
+
+export default function Login() {
+
+    //error slide up
+    const [open, setOpen] = React.useState(false);
+    const [transition, setTransition] = React.useState(undefined);
+
+
+    const state = {
+        email: "",
+        password: "",
+        confirmPassword: "",
+        firstName: "",
+        lastName: "",
+        isLogin: true
+    };
+
+    const resetState = () => {
+        state.email = ""
+        state.password = ""
+        state.confirmPassword = ""
+        state.firstName = ""
+        state.lastName = ""
+    }
```

Comment:
This function also has to clear the text in the textboxes

Reply:
Fixed